bin/tests/otomi.bats

```diff
@@ -21,7 +21,8 @@ function teardown () {
 #############
 @test "otomi without arguments prints help message" {
     run bin/otomi
-    assert_failure 2
+    # Exit code is different depending on global $CI ...
```

Comment:
may I ask why?

Reply:
At the moment, this line is caught by the CI runner and exits 1:

```
[ $has_docker = 'false' ] && [ -z "$CI" ] && err "Otomi CLI expects a running docker daemon!" && exit 1
```

Reply:
Ah, ok. But this test title suggests a message is to be seen. Is that tested for? It seems it only asserts an exit with a code of 1 or 2

Reply:
no answer here? lets move on then...